lib/user_agent/browsers/ok_http.rb

```diff
@@ -59,7 +70,7 @@ def os
       #
       # @return [String]
       def version
-        detect_product(OKHTTP_PRODUCT).version
+        detect { |useragent| OKHTTP_REGEX.match?(useragent.product) }.version
```

Comment:
could this not return nil? Would it make sense to have `detect_produc` support a regular expression?

Reply:
This is the same check as in `extend?` so would never be nil otherwise wouldn't be used as the browser.

Reply:
ok, that's fine by me then